From 900b79ed77cd5c126cc4ac9beef488a94ef57b06 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Sat, 3 Oct 2020 14:05:48 +0200
Subject: [PATCH] ensure that native gendict build by libreoffice is used
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 i18npool/CustomTarget_breakiterator.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/i18npool/CustomTarget_breakiterator.mk b/i18npool/CustomTarget_breakiterator.mk
index dee46a3..fab7ddf 100644
--- a/i18npool/CustomTarget_breakiterator.mk
+++ b/i18npool/CustomTarget_breakiterator.mk
@@ -23,7 +23,7 @@ $(i18npool_BIDIR)/dict_%.data : \
 	$(call gb_Output_announce,$(subst $(WORKDIR)/,,$@),$(true),DIC,1)
 	$(call gb_Trace_StartRange,$(subst $(WORKDIR)/,,$@),DIC)
 	$(call gb_Helper_abbreviate_dirs,\
-		$(call gb_Helper_execute,gendict) $< $@ $(patsubst $(i18npool_BIDIR)/dict_%.cxx,%,$@))
+		$(call gb_Helper_execute,gendict_libre) $< $@ $(patsubst $(i18npool_BIDIR)/dict_%.cxx,%,$@))
 	$(call gb_Trace_EndRange,$(subst $(WORKDIR)/,,$@),DIC)
 
 else # !iOS ANDROID
@@ -38,7 +38,7 @@ $(i18npool_BIDIR)/dict_%.cxx : \
 	$(call gb_Output_announce,$(subst $(WORKDIR)/,,$@),$(true),DIC,1)
 	$(call gb_Trace_StartRange,$(subst $(WORKDIR)/,,$@),DIC)
 	$(call gb_Helper_abbreviate_dirs,\
-		$(call gb_Helper_execute,gendict) $< $@ $(patsubst $(i18npool_BIDIR)/dict_%.cxx,%,$@))
+		$(call gb_Helper_execute,gendict_libre) $< $@ $(patsubst $(i18npool_BIDIR)/dict_%.cxx,%,$@))
 	$(call gb_Trace_EndRange,$(subst $(WORKDIR)/,,$@),DIC)
 
 endif
-- 
2.26.2

