From ecbbe1230e6cc2558cf0cacecb380d990b1aa65f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Andreas=20M=C3=BCller?= <schnitzeltony@gmail.com>
Date: Thu, 19 Aug 2021 15:35:53 +0200
Subject: [PATCH] Do not check / download lib unnecessary for native build
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Andreas Müller <schnitzeltony@gmail.com>
---
 configure.ac | 52 ----------------------------------------------------
 1 file changed, 52 deletions(-)

diff --git a/configure.ac b/configure.ac
index d83c2db..4d780c9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -9353,58 +9353,6 @@ else
 fi
 AC_SUBST([ENABLE_EOT])
 
-dnl ===================================================================
-dnl Check for DLP libs
-dnl ===================================================================
-AS_IF([test "$COM" = "MSC"],
-      [librevenge_libdir="${WORKDIR}/LinkTarget/Library"],
-      [librevenge_libdir="${WORKDIR}/UnpackedTarball/librevenge/src/lib/.libs"]
-)
-libo_CHECK_SYSTEM_MODULE([librevenge],[REVENGE],[librevenge-0.0 >= 0.0.1],["-I${WORKDIR}/UnpackedTarball/librevenge/inc"],["-L${librevenge_libdir} -lrevenge-0.0"])
-
-libo_CHECK_SYSTEM_MODULE([libodfgen],[ODFGEN],[libodfgen-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libepubgen],[EPUBGEN],[libepubgen-0.1])
-
-AS_IF([test "$COM" = "MSC"],
-      [libwpd_libdir="${WORKDIR}/LinkTarget/Library"],
-      [libwpd_libdir="${WORKDIR}/UnpackedTarball/libwpd/src/lib/.libs"]
-)
-libo_CHECK_SYSTEM_MODULE([libwpd],[WPD],[libwpd-0.10],["-I${WORKDIR}/UnpackedTarball/libwpd/inc"],["-L${libwpd_libdir} -lwpd-0.10"])
-
-libo_CHECK_SYSTEM_MODULE([libwpg],[WPG],[libwpg-0.3])
-
-libo_CHECK_SYSTEM_MODULE([libwps],[WPS],[libwps-0.4])
-libo_PKG_VERSION([WPS], [libwps-0.4], [0.4.12])
-
-libo_CHECK_SYSTEM_MODULE([libvisio],[VISIO],[libvisio-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libcdr],[CDR],[libcdr-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libmspub],[MSPUB],[libmspub-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libmwaw],[MWAW],[libmwaw-0.3 >= 0.3.1])
-libo_PKG_VERSION([MWAW], [libmwaw-0.3], [0.3.21])
-
-libo_CHECK_SYSTEM_MODULE([libetonyek],[ETONYEK],[libetonyek-0.1])
-libo_PKG_VERSION([ETONYEK], [libetonyek-0.1], [0.1.10])
-
-libo_CHECK_SYSTEM_MODULE([libfreehand],[FREEHAND],[libfreehand-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libebook],[EBOOK],[libe-book-0.1])
-libo_PKG_VERSION([EBOOK], [libe-book-0.1], [0.1.2])
-
-libo_CHECK_SYSTEM_MODULE([libabw],[ABW],[libabw-0.1])
-
-libo_CHECK_SYSTEM_MODULE([libpagemaker],[PAGEMAKER],[libpagemaker-0.0])
-
-libo_CHECK_SYSTEM_MODULE([libqxp],[QXP],[libqxp-0.0])
-
-libo_CHECK_SYSTEM_MODULE([libzmf],[ZMF],[libzmf-0.0])
-
-libo_CHECK_SYSTEM_MODULE([libstaroffice],[STAROFFICE],[libstaroffice-0.0])
-libo_PKG_VERSION([STAROFFICE], [libstaroffice-0.0], [0.0.7])
-
 dnl ===================================================================
 dnl Check for system lcms2
 dnl ===================================================================
-- 
2.34.1

